packages/camera/camera_android/android/build.gradle

```diff
@@ -34,7 +34,6 @@ android {
     compileSdkVersion 33
 
     defaultConfig {
-        targetSdkVersion 31
```

Comment:
I dont think you want to delete this but instead set it to 33

Reply:
No other plugin in the entire repository has this, and in some initial investigation I was not able to determine exactly how this interacts with app `targetSdkVersion`. If we want the plugins to have this, I think we should have a policy for what we set it to and how we test it, and tooling that enforces that (since if it does affect apps, we need to ensure that we are actually testing, in some cases manually, before we roll it forward).

Reply:
(For instance, does setting this change the compat behavior of every app that is a client of the plugin? If so, updating it to 33 is potentially a big deal for clients.)

Reply:
Filed https://github.com/flutter/flutter/issues/126828 for us to track and discuss this.